src/components/carousel/carousel.test.js

```diff
@@ -0,0 +1,229 @@
+            Slide 5
+          </Typography>
+        </Box>
+      </Slide>
+    </Carousel>
+  );
+};
+
+context("Testing Carousel component", () => {
+  describe("Should render Carousel component", () => {
+    it.each([
+      ["0", "Slide 1"],
+      ["1", "Full clickable slide"],
+      ["2", "Slide 3"],
+      ["3", "Slide 4"],
+      ["4", "Slide 5"],
+    ])("should verify slide %s title is %s", (index, title) => {
+      CypressMountWithProviders(<CarouselComponent />);
+
+      slide(index).should("have.text", title);
+    });
+
+    it.each([
+      ["1", "0", "Slide 1"],
+      ["2", "1", "Full clickable slide"],
+      ["3", "2", "Slide 3"],
+      ["4", "3", "Slide 4"],
+    ])(
+      "when left arrow is clicked to slide %s from slide %s title should be %s",
+      (index, indexSlide, title) => {
+        CypressMountWithProviders(<CarouselComponent slideIndex={index} />);
+
+        clickCarouselButton("left");
+
+        slide(indexSlide).should("have.text", title);
+      }
+    );
+
+    it.each([
+      ["0", "1", "Full clickable slide"],
+      ["1", "2", "Slide 3"],
+      ["2", "3", "Slide 4"],
+      ["3", "4", "Slide 5"],
+    ])(
+      "when right arrow is clicked from slide %s to slide %s title should be %s",
+      (index, indexSlide, title) => {
+        CypressMountWithProviders(<CarouselComponent />);
+
+        clickCarouselButton("right");
+
+        slide(indexSlide).should("have.text", title);
+      }
+    );
+
+    it("should verify left arrow button is disabled on first slide", () => {
+      CypressMountWithProviders(<CarouselComponent slideIndex={0} />);
+
+      previousArrowButton().should("be.disabled").and("have.attr", "disabled");
+    });
+
+    it("should verify right arrow button is disabled on last slide", () => {
+      CypressMountWithProviders(<CarouselComponent slideIndex={4} />);
+
+      nextArrowButton().should("be.disabled").and("have.attr", "disabled");
+    });
+
+    it.each([[true], [false]])(
+      "should verify slide Selector is enabled/disabled",
+      (isEnabled) => {
+        CypressMountWithProviders(
+          <CarouselComponent enableSlideSelector={isEnabled} />
+        );
+
+        if (isEnabled === false) {
+          slideSelector().should("not.exist");
+        } else {
+          slideSelector().should("exist");
+        }
+      }
+    );
+
+    it.each([[true], [false]])(
+      "should verify left arrow button is enabled/disabled",
+      (isEnabled) => {
+        CypressMountWithProviders(
+          <CarouselComponent enablePreviousButton={isEnabled} slideIndex={2} />
+        );
+
+        if (isEnabled === false) {
+          previousArrowButton().should("not.exist");
+        } else {
+          previousArrowButton()
+            .should("have.attr", "type", "button")
+            .and("not.have.attr", "disabled-type", "button");
+        }
+      }
+    );
+
+    it.each([[true], [false]])(
+      "should verify right arrow button is enabled/disabled %s",
```

Comment:
As above, is there any way you can reword this so each case explicitly states what it is testing, like include enabled or disabled in the array, otherwise when it runs in the Cypress dashboard it looks like two instances of "should verify right arrow button is enabled/disabled".